src/checker/text_import.cpp

```diff
@@ -224,6 +226,10 @@ struct text_importer {
         } else {
             throw exception(sstream() << "unknown command: " << cmd);
         }
+        in >> std::ws;
```

Comment:
On MacOS, this line (`in >> std::ws`;) is causing `in.fail()` to flip to `true`.

Reply:
is `in.eof()` true before the call to `std::ws`?